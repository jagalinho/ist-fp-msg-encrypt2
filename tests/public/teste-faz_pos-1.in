pos = faz_pos(0,-5)
pos
