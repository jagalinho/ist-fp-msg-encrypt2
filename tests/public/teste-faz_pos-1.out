'ERRO: ValueError faz_pos'
